.pre-commit-config.yaml

```diff
@@ -0,0 +1,56 @@
+repos:
+- repo: local
```

Comment:
I liked the metadata.yaml validation that max added in k-apps: https://github.com/mesosphere/kommander-applications/pull/127/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R5-R11 can we add the same here?

Reply:
for sure, added in 9467b06fb7032f7c92b30592d0c1a2b192c4b681